.kitchen.yml

```diff
@@ -26,7 +26,12 @@ suites:
     verifier:
       inspec_tests:
         - test/smoke/default
-    attributes:
+  - name: docker
```

Comment:
This needs to be added to .kitchen.yml.erb as well. The .kitchen.yml file is used for local testing with vagrant. The .kitchen.yml.erb uses the ec2 driver to run the tests on aws.

Reply:
updated kitchen.yml.erb with changes from kitichen.yml in a4a8c90